src/MongoDB.Driver/MapReduceOptions.cs

```diff
@@ -177,12 +178,25 @@ public static MapReduceOutputOptions Inline
         /// <param name="sharded">Whether the output collection should be sharded.</param>
         /// <param name="nonAtomic">Whether the server should not lock the database for the duration of the merge.</param>
         /// <returns>A merge map-reduce output options.</returns>
+        [Obsolete("Use a Merge that does not have sharded and nonAtomic parameters instead.")]
         public static MapReduceOutputOptions Merge(string collectionName, string databaseName = null, bool? sharded = null, bool? nonAtomic = null)
         {
             Ensure.IsNotNull(collectionName, nameof(collectionName));
             return new CollectionOutput(collectionName, Core.Operations.MapReduceOutputMode.Merge, databaseName, sharded, nonAtomic);
         }
 
+        /// <summary>
+        /// A merge map-reduce output options.
+        /// </summary>
+        /// <param name="collectionName">The name of the collection.</param>
+        /// <param name="databaseName">The name of the database.</param>
+        /// <returns>A merge map-reduce output options.</returns>
+        public static MapReduceOutputOptions Merge(string collectionName, string databaseName = null)
+        {
+            Ensure.IsNotNull(collectionName, nameof(collectionName));
+            return new CollectionOutput(collectionName, Core.Operations.MapReduceOutputMode.Merge, databaseName, sharded: null, nonAtomic: null);
```

Comment:
Can be simplified to:

    return new CollectionOutput(collectionName, Core.Operations.MapReduceOutputMode.Merge, databaseName, sharded, nonAtomic);